src/tests/CMakeLists.txt

```diff
@@ -12,7 +12,6 @@ foreach(test ${UNIT_TESTS})
 
     if(CUDA_FOUND)
       target_link_libraries("run_${test}" marian marian_cuda ${EXT_LIBS} Catch)
```

Comment:
Does this mean the tests are now part of the repo (submodule or not)?

Reply:
These are unit tests, not regression tests. They are enabled with options. Only check for a number of core functionalities.